### ISSUE TITLE
Vague error when using unsupported stack

### ISSUE THREAD

**Issue by kardolus** [buildpack/pack#103]
We forgot to use our cflinuxfs3-builder and encountered this kind of vague error (using the default bionic builder). 

```
± zm+gk |cnb_wip {2} ✓| → .bin/pack build phpweb --buildpack /tmp/php-cnb-83d544ccc223c057d2bf80d3 --buildpack /tmp/httpd-cnb-83d544ccc223c057d2bf80d3 --buildpack /tmp/php-web-cnb-83d544ccc223c057d2bf80d3 -p integration/fixtures/simple_app
Using default builder image packs/samples:v3alpha2
Pulling builder image packs/samples:v3alpha2 (use --no-pull flag to skip this step)
Selected run image packs/run:v3alpha2 from stack io.buildpacks.stacks.bionic
Pulling run image packs/run:v3alpha2 (use --no-pull flag to skip this step)
Using cache volume pack-cache-744b6d30ed7a8ef4a26a9f736f2a5c09
===> DETECTING
Using manually-provided group
[detector] 2019/02/28 20:02:05 Trying group of 3...
[detector] 2019/02/28 20:02:05 ======== Output: Apache HTTP Server Buildpack ========
[detector]        PHP is in the buildplan, so preparing to serve PHP.
[detector] 2019/02/28 20:02:05 ======== Results ========
[detector] 2019/02/28 20:02:05 PHP Buildpack: pass
[detector] 2019/02/28 20:02:05 Apache HTTP Server Buildpack: pass
[detector] 2019/02/28 20:02:05 PHP Web Buildpack: pass
[detector] 2019/02/28 20:02:05 Error: failed to write buildpack group: open /workspace/group.toml: permission denied
ERROR: run detect container: failed with status code: 1
```

pack version: v0.0.9 (git sha: a1a1a0eef63bd09136ab76663bdbc3b0ab3a4931)

**Comment by ndon55555** [comment]
This also happens when attempting to run `pack build` from inside of another container, even when creating, and then specifying the `org.cloudfoundry.stacks.cflinuxfs3` stack.

**Comment by ekcasey** [comment]
@ndon55555 I believe the issue you were experiencing was fixed in `v0.1.0` (`pack create-builder` now ensures that the files and directories it creates have the correct permissions

@kardolus your original problem looks like a ownership problem w/ the cache volume which also should have been fixed by the new caching model shipped in `v0.1.0`

I'm going to close this. Please feel free to reopen if either of you are still experiencing these issues :)
